### PR TITLE
Add support for AEPEX

### PR DESCRIPTION
**Pull request by git-addrian**
Per discussion [here](https://github.com/daniestevez/gr-satellites/discussions/673), adding gr-satellites support for the Atmospheric Effects of Precipitation through Energetic X-Rays (AEPEX) mission. The decoder created is for 70cm telemetry.

### CODE REVIEW

File: python/telemetry/aepex_sw_stat.py [hunk 1]
```diff
@@ -0,0 +1,446 @@
...
+    'sw_cmd_xsum_state' / Enum(BitsInteger(8), DIS=0, ENA=1),
+    Padding(5),
+    'sw_cmd_arm_state_uhf' / Enum(BitsInteger(1), OFF=0, ARMED=1),
+    'sw_cmd_arm_state_seq' / Enum(BitsInteger(1), OFF=0, ARMED=1),
+    'sw_cmd_arm_state_dbg' / Enum(BitsInteger(1), OFF=0, ARMED=1),
+)
+
+sw_eps_aliveness = BitStruct(
+    Padding(1),
+    'sw_eps_pwr_state_inst4' / Enum(BitsInteger(1), OFF=0, ON=1),
```
Comment (daniestevez) [inline python/telemetry/aepex_sw_stat.py:37]: Same remark regarding code repetition.

File: python/telemetry/aepex_sw_stat.py [hunk 2]
```diff
@@ -0,0 +1,446 @@
...
+    'sw_store_partition_read_log' / BitsInteger(32),
+    'sw_store_partition_write_sci' / BitsInteger(32),
+    'sw_store_partition_read_sci' / BitsInteger(32),
+)
+
+sw_fp = BitStruct(
+    'sw_fp_resp_count' / BitsInteger(16),
+)
+
+sw_ana = BitStruct(
```
Comment (daniestevez) [inline python/telemetry/aepex_sw_stat.py:272]: `LinearAdapter` or `AffineAdapter` is suitable for many of these. A variable can be defined for the adapters of several fields that use the same coefficients (example: `PolynomialAdapter([0.000000e+00, 8.056600e-04], BitsInteger(16))`).

File: python/telemetry/aepex_sw_stat.py [hunk 3]
```diff
@@ -0,0 +1,446 @@
...
+    'sw_ana_afire_curr' / PolynomialAdapter([0.000000e+00, 1.220700e-03],
+                                            BitsInteger(16)),
+    'sw_ana_ifb_therm1' / PolynomialAdapter([1.255500e+02, -1.362200e-01,
+                                             9.861100e-05, -4.417600e-08,
+                                             1.012500e-11, -9.390500e-16],
+                                            BitsInteger(16)
+                                            ),
+)
+
+sw_adcs = BitStruct(
```
Comment (daniestevez) [inline python/telemetry/aepex_sw_stat.py:356]: Same comment about code repetition and adapters.

**Review by daniestevez — CHANGES_REQUESTED** [review]
Thanks for submitting this PR. I have a few comments about coding style.

**Comment by git-addrian** [comment]
Hey @daniestevez , I have made the requested changes. Some notes:

- In order to use LinearAdapter or AffineAdapter, I had to modify the coefficients. For example, if a previous version used PolynomialAdapter([0.0, c1], BitsInteger(8)), it had to be updated to be LinearAdapter(1/c1, BitsInteger(8)). I have gone through and updated each one to reflect it. 
- Packets without APID 0x01 are now handled by GreedyBytes. APID 0x01 is the packet that will beacon. At this time we do not intend to create a decoder for other APIDs. 
- Other feedback has been incorporated!

Best,
Adrian

**Comment by daniestevez** [comment]
Thanks @git-addrian! The code looks good now. However CI is failing. The error is [ModuleNotFoundError: No module named 'satellites'](https://github.com/daniestevez/gr-satellites/actions/runs/11580821781/job/32240195708?pr=675#step:4:513). Sometimes I've seen Python errors in the `satellites` module cause this `ModuleNotFoundError` because of how it is imported within a `try` statement. The real error gets eaten up, which doesn't make the cause obvious.

Can you take at look at this? If this is working fine for you locally, I can take a look too. Maybe the CI pipeline is broken and would also fail for the `main` branch.

**Comment by daniestevez** [comment]
I found the problem by building locally and importing the `python` module from the `build` directory:
```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/daniel/gr-satellites/python/__init__.py", line 54, in <module>
    from .adsb_kml import adsb_kml
  File "/home/daniel/gr-satellites/python/adsb_kml.py", line 15, in <module>
    from .telemetry import gomx_3 as tlm
  File "/home/daniel/gr-satellites/python/telemetry/__init__.py", line 22, in <module>
    from .aepex_70cm import aepex_70cm
  File "/home/daniel/gr-satellites/python/telemetry/aepex_70cm.py", line 18, in <module>
    from .aepex_sw_stat import aepex_sw_stat
  File "/home/daniel/gr-satellites/python/telemetry/aepex_sw_stat.py", line 13, in <module>
    from .adapters import PolynomialAdapter, LinearAdapter
ModuleNotFoundError: No module named 'python.telemetry.adapters'
```

The problem is that this line
```python
from .adapters import PolynomialAdapter, LinearAdapter
```
should say
```python
from ..adapters import PolynomialAdapter, LinearAdapter
```

Can you fix this and test locally that the telemetry parser runs correctly afterwards (since there are no unit tests for it)?

**Comment by git-addrian** [comment]
Hey @daniestevez ,
I was able to fix that line, build it, and confirm it was working with some different test telemetry. I also fixed the following:

- AEPEX.yml was not pointing to its decoder, 'aepex_70cm'. This was silly, and was causing most of my issues with the local build.
- I recieved corrections on which telemetry was supposed to be signed instead of unsigned. Some telemetry readings are now much more accurate.

Please let me know if there are any other changes to be made!

**Comment by daniestevez** [comment]
> AEPEX.yml was not pointing to its decoder, 'aepex_70cm'.

Good point. I didn't notice this.

**Review by daniestevez — APPROVED** [review]
Everything looks good now! Merging.

**Pull request merged**